tapdb/universe.go

```diff
@@ -300,8 +300,6 @@ func upsertAssetGen(ctx context.Context, db UpsertAssetStore,
 		return 0, fmt.Errorf("unable to anchor genesis tx: %w", err)
 	}
 
-	// TODO(roasbeef): need to mark that this is a imported gen?
```

Comment:
I read this TODO as: now that there is no difference between an imported and local genesis record, should we mark the imported genesis as having been imported?

And in that sense I think the TODO should stay.

Reply:
TBF we mark script keys as local vs. imported, so we don't lose the ability to distinguish asset locality.